res/openvpn-gui-res-fr.rc

```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
 
     IDS_NFO_AUTH_PASS_RETRY "Nom d'utilisateur ou mot de passe incorrect. Réessayer..."
     IDS_NFO_KEY_PASS_RETRY  "Mauvais mot de passe. Réessayer..."
-    IDS_ERR_INVALID_PASSWORD_INPUT "Invalid character in password"
-    IDS_ERR_INVALID_USERNAME_INPUT "Invalid character in username"
-    IDS_NFO_AUTO_CONNECT    "Connecting automatically in %u seconds…"
-    IDS_NFO_CLICK_HERE_TO_START "OpenVPN GUI is already running. Right click on the tray icon to start."
-    IDS_NFO_BYTECOUNT "Bytes in: %ls  out: %ls"
+    IDS_ERR_INVALID_PASSWORD_INPUT "Caractère invalide dans le mot de passe"
+    IDS_ERR_INVALID_USERNAME_INPUT "Caractère invalide dans le nom d'utilisateur"
+    IDS_NFO_AUTO_CONNECT    "Connexion automatique dans %u secondes…"
```

Comment:
I would probably think that 'connecter automatiquement ...' is better phrase to use